man/rgbgfx.1

```diff
@@ -248,8 +248,7 @@ See
 below for details.
 .Pp
 .Ar width
-is the image's width, in tiles
-.Pq including any margins specified by Fl L .
+is the image's width, in tiles.
```

Comment:
```suggestion
is the width of the area to process
.Pq thus excluding any margins specified with Fl L ,
in tiles.
```

Reply:
`-r` neither excludes nor includes margins from `-L` because `-L` may not coexist with `-r`. Also "margin" is not used to describe how `-L` works.

Reply:
```suggestion
is the width of the image to generate, in tiles.
```

Is this better?

Reply:
Yes, thanks.